src/promptflow-devkit/promptflow/_cli/_utils.py

```diff
@@ -442,8 +442,8 @@ def get_instance_results(path: Union[str, Path]) -> List[Dict]:
             for line in f:
                 data = json.loads(line)
                 run_info = data.get("run_info", {})
-                inputs = run_info.get("inputs", {})
-                output = run_info.get("output", {})
+                inputs = run_info.get("inputs", None) or {}
+                output = run_info.get("output", None) or {}  # output can be None for some cases
```

Comment:
what's the difference between this two?
run_info.get("inputs", None) or {}
run_info.get("inputs", {})

Reply:
This is the strange error I met yesterday, turns out for prompty run, run_info has "output": null, so here output will be None and fail at line 452